aaronetwork/chain.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "aaronetwork",
```

Comment:
I think this chain_name will lead to a lot of confusion. If keeping the 'network' part, then I'd suggest "aaronnetwork" instead.

Reply:
Btw, this suggestion is not a requirement. I see that the merging of the two 'n's is used in many places (chain_id, website, twitter, explorer url, etc.)

Reply:
Should chain_name be fused? Can we specify "aaron network" or only "aaronnetwork"?

Reply:
Space characters are not allowed. This is just for identifying, and never shown to app users.